src/content/docs/en/guides/integrations-guide/tailwind.mdx

```diff
@@ -148,9 +148,9 @@ You can now [import your own `base.css` as a local stylesheet](/en/guides/stylin
 ## Examples
 
 *   The [Astro Tailwind Starter](https://github.com/withastro/astro/tree/latest/examples/with-tailwindcss?on=github) gets you up and running with a base for your project that uses Tailwind for styling
-*   Astro's homepage uses Tailwind. Check out its [Tailwind configuration file](https://github.com/withastro/astro.build/blob/main/tailwind.config.cjs) or an [example component](https://github.com/withastro/astro.build/blob/main/src/components/IntegrationCard.astro)
+*   Astro's homepage uses Tailwind. Check out its [Tailwind configuration file](https://github.com/withastro/astro.build/blob/main/tailwind.config.cjs) or an [example component](https://github.com/withastro/astro.build/blob/main/src/components/Header.astro)
```

Comment:
```suggestion
*   Astro's homepage uses Tailwind. Check out its [Tailwind configuration file](https://github.com/withastro/astro.build/blob/main/tailwind.config.cjs) or an [example component](https://github.com/withastro/astro.build/blob/main/src/components/Checkbox.astro)
```

I think it might be more valuable to include another component such as `Checkbox.astro` where there's a mixed use of `<style>` tags with `@apply` classes and general classes in HTML